src/ifp/src/InitFloorplan.cc

```diff
@@ -225,7 +229,7 @@ void InitFloorplan::initFloorplan(
       if (row_parity != RowParity::NONE) {
         logger_->error(
             IFP,
-            41,
+            51,  // old was 41
```

Comment:
no need to comment what the old error number was. The message docs are regenerated automatically